Makefile.kubebuilder

```diff
@@ -0,0 +1,129 @@
+# Copyright 2019 The Kanister Authors.
+#
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Image URL to use all building/pushing image targets
+
+REPOSITORY_SERVER_IMAGE_NAME := repo-server-controller
+IMG ?= $(REGISTRY)/$(REPOSITORY_SERVER_IMAGE_NAME)
```

Comment:
To avoid confusion with Kanister controller IMAGE https://github.com/kanisterio/kanister/blob/master/Makefile#L59 
```suggestion
REPOSERVER_IMAGE ?= $(REGISTRY)/$(REPOSITORY_SERVER_IMAGE_NAME)
```